src/StickerService.ts

```diff
@@ -41,7 +41,10 @@ function buildWallpaperCss({
   .monaco-workbench .part.editor>.content .editor-group-container>.title .tabs-breadcrumbs .breadcrumbs-control,
   .ref-tree, /* find usages */
   .head, /* find usages */
-  .monaco-workbench .part.editor>.content .editor-group-container>.title .editor-actions
+  .monaco-workbench .part.editor>.content .editor-group-container>.title .editor-actions,
+  .editor-container, /* welcome screen */
+  .terminal-outer-container,  /* Terminal outer edge */
```

Comment:
What was the point of `terminal-outer-container`?
It makes it look a bit off.

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/15972415/114384883-e8c95980-9b54-11eb-8eb8-4d3cf9ec7e62.png) | ![image](https://user-images.githubusercontent.com/15972415/114384810-d51df300-9b54-11eb-837a-99dec750ffbe.png) |

Reply:
Hmm, Looks like there may be differences between VS Code on various operating systems.

I've fortunately (and sometimes unfortunately) use a Linux, MacOS, and Windows 10 machine. I'll have to take a peek to see if Windows is being a jerk.

Reply:
Ah, with the most recent changes, the bright ring around the terminal went away :tada: